packages/frontend/src/components/TokenWrapper/TokenWrapperContext.tsx

```diff
@@ -158,7 +160,7 @@ const TokenWrapperContextProvider: FC = ({ children }) => {
 
   const unwrap = async () => {
     try {
-      const networkId = Number(selectedNetwork.networkId)
+      const networkId = Number(selectedNetwork?.networkId)
```

Comment:
if there's no `selectedNetwork.networkId` then this function should `return`

Reply:
i believe this would get resolved in the subsequent [check](https://github.com/hop-protocol/hop/blob/5200f96df36739113a96d20a14fbee098e97aed0/packages/frontend/src/contexts/Web3Context.tsx#L282), but i'll add a change here for readability